WorkAround #2179 (#3226)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/windows-defender-exploit-guard/customize-exploit-protection.md

>The result will be that DEP will be enabled for *test.exe*. DEP will not be enabled for any other app, including *miles.exe*. 
>CFG will be enabled for *miles.exe*.

> >[!NOTE] If you found any issues on this article you can report it directly to a Windows Server/Windows Client partner or use our technical support numbers on your country.

Reviewer: Just to make it even more visible:
```suggestion
>[!NOTE]
>If you have found any issues in this article, you can report it directly to a Windows Server/Windows Client partner or use the Microsoft technical support numbers for your country.
```

> 

Reviewer: If the added text will be accepted, you should probably rephrase the following:
> ... or use **the Microsoft** technical support numbers **for** your country.

Reviewer: Also, code style adjustment needed: add a NewLine after `>[!NOTE]` as used in the other notes, warnings and important notices elsewhere in the docs.
```
>[!NOTE]
>If you found any issues on this article, you can report it directly to a Windows Server/Windows Client partner or use the Microsoft technical support numbers for your country.
```

Reviewer 2: Agree with copy as edited by illfated.

See the [Windows Security](../windows-defender-security-center/windows-defender-security-center.md#customize-notifications-from-the-windows-defender-security-center) topic for more information about customizing the notification when a rule is triggered and blocks an app or file.

 

> 

Reviewer: These 2 empty blank lines are redundant. Please remove.

Author: Done
